deployer/loader.go

```diff
@@ -81,7 +80,7 @@ func (l *StateLoader) LoadGatewayNameFromGrid(nodeID uint32, name string) (workl
 
 // LoadZdbFromGrid loads a zdb from grid
 func (l *StateLoader) LoadZdbFromGrid(nodeID uint32, name string) (workloads.ZDB, error) {
-	wl, err := l.GetWorkload(nodeID, name)
+	wl, _, err := l.GetWorkloadInDeployment(nodeID, name)
```

Comment:
will the name stay unique? now, that we lifted the limitation of having one deployment on the node? because now you can have multiple deployments and the names can be reused within different deployments (except for the network)